terraform/network/aws-vpc/main.tf

```diff
@@ -0,0 +1,311 @@
+// Define the required Terraform version and providers
+terraform {
+  required_version = ">=1.8"
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 5.0"
```

Comment:
set this to a full major.minor.patch semver, and usually set it to strict equals. This way we'll let renovate handle patching it.